lib/jnpr/junos/facts/get_chassis_cluster_status.py

```diff
@@ -17,20 +31,45 @@ def get_facts(device):
     """
     srx_cluster = None
     srx_cluster_id = None
+    redundancy_group = None
 
     try:
-        rsp = device.rpc.get_chassis_cluster_status(normalize=True,
-                                                    redundancy_group="0")
+        rsp = device.rpc.get_chassis_cluster_status(normalize=True)
```

Comment:
If by this time if we already have the facts with personality details, shouldnt we use that to check if the device type is SRX (which is not a major percentage of our products in customer place), hence saving unnecessary RPC calls to different type of devices.